sql/server.go

```diff
@@ -570,7 +570,7 @@ func (s *Server) Set(session *Session, p *parser.Set, args []driver.Datum,
 	// be double quoted.
 	name := strings.ToLower(p.Name.String())
 	switch name {
-	case `"database"`: // Quoted: database is a reserved word
+	case `"database"`, `database`:
```

Comment:
ok, I can change expr.go to lookup keywords in a case insensitive way. I was just wondering if there was a reason for the way things are right now